### DISCUSSION TITLE
Is the open source software subject to US Export Administration Regulations

### DISCUSSION THREAD

**Discussion by spfyoung** [community/community#127565] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a question, whether the open source code on Github (just for a example [react-native](https://github.com/facebook/react-native)) is affected by the US Export Administration Regulations (EAR), whether it is allowed to be used in sanctioned countries such as Iran or Cuba, and whether it can be run on Huawei phones (Huawei is on the US Entity List). 
Need some evidence to convince the legal guy. thanks.

**Comment by hamburnyog** [comment]
Open source software binaries that are published publicly are not subject to the EAR.

However if you take that code and build something new, with extra features or encryption, that new app isn't automatically free from rules.

You can read more here:

https://www.linuxfoundation.org/resources/publications/understanding-us-export-controls-with-open-source-projects

https://www.bis.doc.gov/index.php/policy-guidance/encryption/1-encryption-items-not-subject-to-the-ear
